### DISCUSSION TITLE
timezone support for date time

### DISCUSSION THREAD

**Discussion by mjnamjkaze** [element-plus/element-plus#16790] [Feature Request]
Hello friend, I see Element plus using dayjs
Recently this lib support setting timezone, you can see here :
https://day.js.org/docs/en/timezone/set-default-timezone
could us have timezone settings for element plus? Same the "locale" in config-provider-props.d.ts ?

**Comment by joycewen0910** [comment]
I neeeeeeeeeeed timezone
